hoot-core-test/src/test/cpp/hoot/core/io/OsmApiDbTest.cpp

```diff
@@ -71,23 +84,36 @@ class OsmApiDbTest : public CppUnit::TestFixture
     }
   }
 
-  QUrl getOsmApiDbUrl()
+  //TODO: the sql output shouldn't be written out to null; simply checking the return value
+  //isn't enough to know whether it failed or not
```

Comment:
It should.  I'm not sure the best way yet.  Its going straight to stdout and will add a lot of logging to the build.  That test had been silently failing for probably months b/c that output was being swallowed.  I guess I could direct the output to its own log file and grep for errors ???  seems kludgy...